frame.go

```diff
@@ -33,6 +33,10 @@ func (f *Frame) AllocBuffer(align int) error {
 	return newError(C.av_frame_get_buffer(f.c, C.int(align)))
 }
 
+func (f *Frame) AllocHWBuffer(hfc *HardwareFrameContext, align int) error {
```

Comment:
I don't think you need `align int` here since the doc states that the final argument of `av_hwframe_get_buffer` is `flags` and is currently unused and should be set to 0